OctopusDSC/DSCResources/cTentacleAgent/cTentacleAgent.psm1

```diff
@@ -51,11 +51,23 @@ function Get-MachineFromOctopusServer
 
         [Parameter()]
         [System.String]
-        $Instance
+        $Instance,
+
+        [Parameter()]
```

Comment:
Can we make `$Instance` and `$SpaceId` mandatory? They are internal functions, and we've been bitten by optional params in the past...